docs/book/platform-guide/set-up-your-mlops-platform/use-the-secret-store/interact-with-secrets.md

```diff
@@ -76,8 +74,7 @@ By default, all created secrets are scoped to the active workspace. To create a
 ```shell
 zenml secret create <SECRET_NAME> \
     --scope user \
-    --<KEY_1>=<VALUE_1> \
-    --<KEY_2>=<VALUE_2>
+    --values='{"key1":"value2","key2":"value2"}'
```

Comment:
same as before: document the new option _in addition to the new one_, not by replacing it.